UNRELEASED_CHANGELOG.md

```diff
@@ -67,10 +67,29 @@
 
 ## stream-chat-android-ui-components
 ### 🐞 Fixed
-
+Fix for position of deleted messages for other users
 ### ⬆️ Improved
 
 ### ✅ Added
+Now it is possible to customize when the avatar appears in the conversation. It is
```

Comment:
One last observation, leaving this comment on a random line so that it can be used for a discussion.

In message options, hiding the avatar makes it so that the option items no longer line up with the message bubble:

| Shown | Hidden |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/12054216/130251806-9875177b-7620-4bce-a0bb-4368a0790dcb.png)  | ![image](https://user-images.githubusercontent.com/12054216/130251816-8582c1a4-d69b-4451-881b-1681a16abb2c.png) |

The design for our/their message options shows the correct avatar visible/invisible handling nicely (as one shows, and one hides the avatar):

![image](https://user-images.githubusercontent.com/12054216/130251921-763510d5-b8c3-4781-bb6f-c79f574f5898.png)

The images above show the problem on _their_ side, but I assume it'd also be misaligned if we showed avatars for our own messages, and then opened message options.

---

Bonus: what's that gap between the message and the options below it? Blank space for the footnote? We should clean that up, but as it exists on `develop` too, it can go into another issue / PR.

Reply:
Good eye. Let me take a look.

Reply:
Done. The margin of the options follows the margin for the message